.github/actions/start-opensearch-with-one-plugin/action.yml

```diff
@@ -10,14 +10,10 @@ inputs:
     description: 'The name of the plugin to use, such as opensearch-security'
     required: true
 
-  plugin-start-script:
-    description: 'The file name for the configuration script for the plugin such as install_demo_configurations -- may not be needed for every plugin'
+  setup-script:
```

Comment:
Please change this parameters value to be the script name such as 'setup.bat' or 'setup.sh', hardcoding where the script is and how its named its difficult to follow